src/m365/planner/commands/tenant/tenant-settings-list.ts

```diff
@@ -0,0 +1,34 @@
+import { Logger } from '../../../../cli';
+import { AxiosRequestConfig } from 'axios';
+import request from '../../../../request';
+import PlannerCommand from '../../../base/PlannerCommand';
+import commands from '../../commands';
+
+class PlannerTenantSettingsListCommand extends PlannerCommand {
```

Comment:
Let's define the `defaultProperties` on the command for when `text` output is used.

```
public defaultProperties(): string[] | undefined {
	return ['id', 'title'];
}
```

Reply:
The example above is an example right? Because the ID property is quite useless and the title property doesn't exist.
Currently the text output generated is quite clear to me. Only thing I could do maybe is define defaultProperties so the id property is omitted. Or is this what you meant?

Current output:

![image](https://user-images.githubusercontent.com/11723921/171856894-58dab4e3-1550-4505-8dd8-aa4f46260286.png)

Reply:
> The code above is an example right?

Yes this was just an example, apologies, I should have explained better 😊

Thank you for the update 👍